tGUI.m

```diff
@@ -74,7 +74,87 @@
 %                      - saved function as tGUI.m 
 % 2017.12.05 - coolbac - updated to version 0.2.2 changes include:
 %		       - corrected errors in version 0.2.1 that did not save during previous commit
-function csv_filename = tGUI(experiment_name_string, gui_params)
+% 2018.07.19 - joseferpaz - updated to version 1.0.0 changes include:
+%                         - compatibility with arduino temperature sensor
+%                         - temperature readings logged into csv file
+%                         - Waits for thermoneutral before running tGUI
+%                     
+%
+%% Initialize arduino
+
+clear all;
+global sensor1_analog sensor1_t sensor2_analog sensor2_t uno;
+global sensor1_port sensor2_port filename;
+
+uno  = arduino();
+  
+%Port for LED
+light_port = 'D3';
+  
+%Port for sensor 1 (Green)
+sensor1_port = 'A5';
+  
+%Port for sensor 2 (Blue)
+sensor2_port = 'A2';
+  
+% make sure LED is off when initialized
+writePWMDutyCycle(uno, light_port , 0);
+  
+%sensor1_data_t = zeros(1);
+%sensor2_data_t = zeros(1);
+%   Arrays for voltage readings from sensors
+%   sensor1_data_v = zeros(1);
+%   sensor2_data_v = zeros(1);
+
+%% Get file name
+filename = input('File name: ','s');
+
+if isempty(filename)
+    filename = '';
+end
+
+
+%% Read the temperature sensors
+
+stopcmd = 0;
+
+while stopcmd == 0  
+      
+    
+    % Get measurements from sensor 1 (green) and print to command window
```

Comment:
Could you please include a comment to describe the calculations used in this code? Where does the 1023/5  come from?

Reply:
@ccoolbaugh , I looked into it and it turns out all constants (except for 100) cancel out between lines 130 & 131.